docs/config/README.md

```diff
@@ -1208,14 +1208,23 @@ The module will be shown if any of the following conditions are met:
 
 ### Options
 
-| Variable             | Default         | Description                                                                 |
-| -------------------- | --------------- | --------------------------------------------------------------------------- |
-| `symbol`             | `"🐍 "`         | The symbol used before displaying the version of Python.                    |
-| `pyenv_version_name` | `false`         | Use pyenv to get Python version                                             |
-| `pyenv_prefix`       | `"pyenv "`      | Prefix before pyenv version display (default display is `pyenv MY_VERSION`) |
-| `scan_for_pyfiles`   | `true`          | If false, Python files in the current directory will not show this module.  |
-| `style`              | `"bold yellow"` | The style for the module.                                                   |
-| `disabled`           | `false`         | Disables the `python` module.                                               |
+| Option               | Default                                                    | Description                                                                 |
+| -------------------- | ---------------------------------------------------------- | --------------------------------------------------------------------------- |
+| `format`             | `"via [${symbol}${version}( \\($virtualenv\\))]($style) "` | The format for the module.                                                  |
+| `symbol`             | `"🐍 "`                                                     | A format string representing the symbol of Python                           |
+| `style`              | `"yellow bold"`                                            | The style for the module.                                                   |
+| `pyenv_version_name` | `false`                                                    | Use pyenv to get Python version                                             |
+| `pyenv_prefix`       | `"pyenv "`                                                 | Prefix before pyenv version display (default display is `pyenv MY_VERSION`) |
```

Comment:
Quickly skimmed through your code, and I didn't find this option used anywhere in your code.

Reply:
True, removed it from this section..